app/slack_api.py

```diff
@@ -0,0 +1,23 @@
+def notify_slack(token, task_name):
+    import requests
+    # import os
+    # from dotenv import load_dotenv
+
+    # load_dotenv()
+
+    # token = os.environ.get("SLACK_BOT_USER_OAUTH_TOKEN")
+    text = f"Someone just completed {task_name}"
+    # auth_header = {"Authorization": f"Bearer {token}"}
+
+    slack_response = requests.post('https://slack.com/api/chat.postMessage', data={"token": token, "channel": "task-notifications", "text": text})
+    # print(response.status_code)
+    print(slack_response.json())
```

Comment:
Printing to the console is a very useful debugging technique, however, we don't want to end up merging code in that contains print statements, so be sure to check your code before submitting a PR to make sure you've removed any debugging print statements.